docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.profile

```diff
@@ -22,7 +22,8 @@ use Drupal\user\RoleInterface;
 function su_humsci_profile_contextual_links_alter(array &$links, $group, array $route_parameters) {
   if ($group == 'paragraph') {
     // Paragraphs edit module clone link does not function correctly. Remove it
-    // from available links.
+    // from available links. Also remove delete to avoid unwanted delete.
+    unset($links['paragraphs_edit.delete_form']);
```

Comment:
These are granted by permissions no? You want the user to be able to delete, just not through a contextual link? Isn't there a confirmation page?

Reply:
yeah. H&S just wanted to avoid the confusion. Partly because you can delete a paragraph row or the paragraph within the row.